db/migrate/20230110144239_create_slack_user_profiles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSlackUserProfiles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :slack_user_profiles do |t|
+      t.string :user_id
```

Comment:
Is this supposed to be for the Double Up `user_id` or the Slack `user_id`? If Slack, it might be worth adding `slack_` as a prefix.

Reply:
Good call - it is for Slack, will refac to `slack_user_id`